components/function-controller/internal/controllers/serverless/function_reconcile_test.go

```diff
@@ -151,10 +146,10 @@ func TestFunctionReconciler_Reconcile_Scaling(t *testing.T) {
 
 		t.Log("updating function to use scalable replicas number")
 		g.Expect(resourceClient.Get(context.TODO(), request.NamespacedName, function)).To(gomega.Succeed())
-		function.Spec.ScaleConfig.MaxReplicas = &four
+		function.Spec.Replicas = &four
```

Comment:
I would change the name of the test step to something like: `external scaler increase replicas amount`